File: benchmarks/arrays/arrays.c
#include <stdio.h>
#define N 10000000

int main() {
    static int arr[N];
    for (int i = 0; i < N; i++) {
        arr[i] = i % 100;
    }
    long sum = 0;
    for (int i = 0; i < N; i++) {
        sum += arr[i];
    }
    printf("%ld\n", sum);
    return 0;
}

File: benchmarks/arrays/arrays.go
package main

import "fmt"

const N = 10000000

func main() {
	arr := make([]int32, N)
	for i := 0; i < N; i++ {
		arr[i] = int32(i % 100)
	}
	var sum int64 = 0
	for i := 0; i < N; i++ {
		sum += int64(arr[i])
	}
	fmt.Println(sum)
}
